src/components/NewCommentForm.tsx

```diff
@@ -14,24 +127,32 @@
             name="name"
             id="comment-author-name"
             placeholder="Name Surname"
-            className="input is-danger"
+            className={classnames('input', {
```

Comment:
Great use of the `classnames` library to conditionally apply classes based on the form state.